docker/prod/README.md

```diff
@@ -1,5 +1,156 @@
 # Production docker
 
-This directory contains Dockerfiles for generating images to be used in **all** deployed environments (as opposed to local development).
+This directory contains Dockerfiles for generating images to be used in all AWS environments (as opposed to local development).
 
-> DOCKERFILES ARE NOT YET COMPLETE
+It is designed to run on AWS, using S3 and SQS (as opposed to file-system storage and RabbitMQ).
+
+The Django settings module must be `panelapp.settings.docker-aws` (`DJANGO_SETTINGS_MODULE=panelapp.settings.docker-aws`).
+
+Different environments (e.g. Staging, Prod) are configured setting the environment variables below.
+
+Most of the configurations are the same for `web` and `worker` containers.
+
+To configure it for running with LocalStack, look at the comments in `panelapp/panleapp/settings/docker-aws.py`.
+
+##  Mandatory environment variables
+
+All of the following must be explicitly configured
+
+### Non-Secrets
+
+* `AWS_S3_STATICFILES_BUCKET_NAME` - Name of the S3 bucket for storing static files - `web` only
+* `AWS_S3_MEDIAFILES_BUCKET_NAME` - Name of the S3 bucket for storing media (uploaded) files
+* `AWS_REGION` - AWS Region
+* `AWS_S3_STATICFILES_CUSTOM_DOMAIN` - Custom CDN domain to serve static files from (e.g. `cdn.mydomain.com` - no trailing `/`) - `web` only
+* `ALLOWED_HOSTS` - whitelisted hostnames, if user tries to access website which isn't here Django will throw 500 error - `web` only
+* `DEFAULT_FROM_EMAIL` - we send emails as this address
+* `PANEL_APP_EMAIL` - PanelApp email address
+* `EMAIL_HOST` - SMTP server hostname
+* `EMAIL_PORT` - SMTP server port
+
+### Secrets
+
+* `DATABASE_URL` - PostgreSQL config url, in the format: `postgresql://{username}:{password}@{host}:{port}/{database_name}`
+* `EMAIL_HOST_USER` - SMTP username (no SMTP authentication if omitted)
+* `EMAIL_HOST_PASSWORD` - SMTP password (no SMTP authentication if omitted)
+* `HEALTH_CHECK_TOKEN` - Token to authenticate health check endpoint requests
+* `SECRET_KEY` - Secret for encrypting cookies
+
+
+## Optional environment variables
+
+### Non-secrets
+
+* `DJANGO_LOG_LEVEL` - to override Django log-level (default=`INFO`)
+* `EMAIL_USE_TLS` - Set to `False` to prevent SMTP from using TLS
+* `SQS_QUEUE_VISIBILITY_TIMEOUT` - SQS topic _Visibility Timeout_. Must be identical to the Visibility Timeout of the SQS queue
+* `TASK_QUEUE_NAME` - Name of the SQS queue (default: `panelapp`)     
+
+### Secrets
+
+* `DJANGO_ADMIN_URL` - change admin URL to something secure - `web` only
```

Comment:
`web` and `workers`, ideally all containers and processes should have exactly the same configuration as it might end up with different result

Reply:
Understood.
I thought Django was not exposing any web interface for `worker`. But keeping the configuration identical is not a problem.
I'm amending docs

Reply:
It doesn't but we might use this setting to construct an admin URL in the worker for example